frontend/public/components/factory/ListPage/ListPageFilter.tsx

```diff
@@ -40,7 +24,6 @@ const ListPageFilter: React.FC<ListPageFilterProps> = ({
       nameFilterPlaceholder={nameFilterPlaceholder}
       labelFilterPlaceholder={labelFilterPlaceholder}
       onFilterChange={onFilterChange}
-      textFilter={textFilter}
```

Comment:
Don't we need this prop passthrough anymore?

Reply:
maybe in the future, but in other form.

`textFilter` is a string name of some function in https://github.com/openshift/console/blob/master/frontend/public/components/factory/table-filters.ts#L39

which is very dangerous.

If in the future we will need to provide custom implementation of text filter a consumer will be able to pass a new optional property as a function, not string. The optional property wont break the API

So we are good for now.